timestampvm/src/api/chain_handlers.rs

```diff
@@ -181,8 +180,9 @@ where
     }
 }
 
-fn create_jsonrpc_error(e: std::io::Error) -> Error {
+fn create_jsonrpc_error<E: Borrow<std::io::Error>>(e: E) -> Error {
```

Comment:
Clippy complained that this function took ownership without needing to take ownership. Using `Borrow` allows either `&std::io::Error` or `std::io::Error` (borrowed or owned respectively) to be used as arguments